src/components/LineNavigator.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC } from 'react';
+import { Box, Group, MediaQuery, createStyles } from '@mantine/core';
+import { IconChevronLeft, IconChevronRight } from '@tabler/icons-react';
+
+const useStyles = createStyles(theme => ({
+  iconWrapper: {
+    borderRadius: '50%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    transition: 'background-color 100ms linear',
+  },
+
+  lightmode: {
+    width: 40,
+    height: 40,
+    border: '1px solid #EEE7E7',
+  },
+
+  darkmode: {
+    backgroundColor: '#292929 !important',
+  },
+
+  big: {
+    [theme.fn.smallerThan('md')]: {
+      width: 55,
+      height: 55,
+    },
+  },
+
+  default: {
+    [theme.fn.smallerThan('md')]: {
+      width: 32,
+      height: 32,
+    },
+  },
+}));
+
+export const LineNavigator: FC<any> = ({
```

Comment:
```suggestion
export const LineNavigator: FC<any> = ({
```
Ini jangan kasih `any`, kasih type props yang lebih proper ya. Coba pakein generic types aja kalo memang butuh types-nya dinamis.